kiva/fonttools/font_manager.py

```diff
@@ -670,6 +671,27 @@ def afmFontProperty(fontpath, font):
 
     return FontEntry(fontpath, name, style, variant, weight, stretch, size)
 
+def extractTTC(fpath):
+    """
+    Extract a list of ttfFontProperty objects from a ttc file
+    """
+    props = []
+    try:
+        collection = TTCollection(str(fpath))
+    except (RuntimeError, TTLibError):
+        verbose.report("Could not open font collection %s" % fpath)
+        return props
+    except UnicodeError:
```

Comment:
Is this problem with Unicode limitation of fonttools, Windows, and/or the particular Python version you are using?

Reply:
No clue. I assume there was some problem with fonttools itself that we are trying to avoid. I see the same check in matplotlib's code. Somewhat moot with 97d9c6a

Reply:
Ah - it's possible that this is no longer relevant.  We used to package our own version of fonttools, but now we use a 3rd party wrapper.  It's fine to leave in here, but it may be that since that switch we no longer will get the error.